templates/navigation-bar.twig

```diff
@@ -1,4 +1,137 @@
 {% block navigation_bar %}
+{% if new_design_navigation_bar %}
```

Comment:
Would be a bit nicer if we had the old and new version in a separate file.

That way, once we remove the old design, we don't need to remove the if statement from around all markup of the new version, which would change indention of all lines.